dev/tests/api-functional/testsuite/Magento/GiftMessage/Api/OrderItemGetRepositoryTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\GiftMessage\Api;
+
+use Magento\TestFramework\TestCase\WebapiAbstract;
+
+class OrderItemGetRepositoryTest extends WebapiAbstract
```

Comment:
Add class description

Reply:
class description to a test ? we dont have this for tests